src/main/java/io/quarkus/search/app/hibernate/AnalysisConfigurer.java

```diff
@@ -16,7 +16,8 @@ public class AnalysisConfigurer implements ElasticsearchAnalysisConfigurer {
     public void configure(ElasticsearchAnalysisConfigurationContext context) {
         context.analyzer(DEFAULT).custom()
                 .tokenizer("standard")
-                .tokenFilters("lowercase", "asciifolding", "stemmer");
+                .tokenFilters("lowercase", "asciifolding", "stemmer")
+                .charFilters("html_strip");
```

Comment:
Can you please add it to `AUTOCOMPLETE` as well? It shouldn't matter much given how little HTML we have in titles/summaries, but might spare us some debugging of inconsistent matches.